Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
 //Write code for the 1st task below this
-
-
+function addTWo(num1,num2){
+    return num1+num2;
```

Comment:
The function name 'greet' should be in camelCase, so it should be 'greet' instead of 'greet'.